main.go

```diff
@@ -103,6 +106,8 @@ func main() {
 	pflag.StringVar(&autoInstrumentationDotNet, "auto-instrumentation-dotnet-image", fmt.Sprintf("ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet:%s", v.AutoInstrumentationDotNet), "The default OpenTelemetry DotNet instrumentation image. This image is used when no image is specified in the CustomResource.")
 	pflag.StringArrayVar(&labelsFilter, "labels", []string{}, "Labels to filter away from propagating onto deploys")
 	pflag.IntVar(&webhookPort, "webhook-port", 9443, "The port the webhook endpoint binds to.")
+	pflag.StringVar(&tlsOpt.minVersion, "tls-min-version", "VersionTLS12", "Minimum TLS version supported. Value must match version names from https://golang.org/pkg/crypto/tls/#pkg-constants.")
```

Comment:
is it guaranteed that `VersionTLS12` is always present?

Reply:
This add the option that default will be set to tlsv1.2, and if user wanna choose other than v1.2, they can do it from the command arg. So we have the flexibility and in case for upgrading, they able to do it from command argument as well.

Reply:
Currently in Jaeger-operator, I m using the same method as well with suggestions from @frzifus. He been helping me on review and give me the idea on configurable tls and cipher settings